### ISSUE TITLE
Жаль что нет поддержки Vue.js 2

### ISSUE THREAD

**Issue by balalamba** [Saymon-biz/vue-localize#5]
Плагин просто супер, просто идеален для быстрого старта мультиязычного сайта. Жаль только что он видимо не совместим с vue 2! Планируешь ли обновлять ?

**Comment by Saymon-biz** [comment]
Да, планируем в ближайшее время адаптировать под Vue 2 + добавить несколько фич, в том числе и по сущесствующим ишьюсам. Старь проект, подписывайся и будешь в курсе обновлений. Также любые соображение по улучшениям much appreciated.

**Comment by dotnatcore** [comment]
any update/plan to update?

**Comment by nobilix** [comment]
hey, @dotnatcore yep, we're about to update the plugin soon, working on vue.js 2 compatibility and on new features. stay updated by subscribing on the repo

**Comment by joakimglaving** [comment]
any news on the update? :)

**Comment by Saymon-biz** [comment]
@bobcat1 yeah! we had performed upgrade to Vue 2 already and this work is in [appropriate branch](https://github.com/vuedev-com/vue-localize/tree/migration-to-v2) yet, for now. Now we plan to add the following features:

- loading translations on demand from JSON API (#1)
- an ability to organize translations per component (inside .vue files) [requested here](https://github.com/vuedev-com/vue-localize/issues/1#issuecomment-234976395)
- an ability to split translations trees by language but provide an ability to support [both approaches](https://github.com/vuedev-com/vue-localize/issues/1#issuecomment-235232034)

And only then we plan to perform testing, update documentation and merge this work into master, and publish to NPM

But there is no any exact date when it will be done

**Comment by joakimglaving** [comment]
@Saymon-biz great! thank you for the updated information
